pom.xml

```diff
@@ -8,26 +8,17 @@
 
   <groupId>com.cloudbees.jenkins.plugins</groupId>
   <artifactId>docker-custom-build-environment</artifactId>
-  <name>CloudBees Docker Custom Build Environment Plugin</name>
+  <name>Docker Custom Build Environment Plugin</name>
   <version>1.6.6-SNAPSHOT</version>
   <packaging>hpi</packaging>
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Custom+Build+Environment+Plugin</url>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/Docker+Custom+Build+Environment+Plugin</url>
 
   <licenses>
     <license>
       <name>MIT</name>
     </license>
   </licenses>
 
-  <developers>
```

Comment:
Would be great to have a `retired` flag just to give credit to original maintainers, but it is not how it works now